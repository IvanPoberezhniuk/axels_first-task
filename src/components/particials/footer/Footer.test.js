import { unmountComponentAtNode } from 'react-dom';

import { Footer } from '../../index';

const setUp = () => render(<Footer />);
let component;

beforeEach(() => {
  component = setUp();
});

afterEach(() => {
  unmountComponentAtNode(component);
  component.remove();
  component = null;
});

describe('Footer component', () => {
  it('SNAPSHOT', () => {
    expect(component).toMatchSnapshot();
  });
});

