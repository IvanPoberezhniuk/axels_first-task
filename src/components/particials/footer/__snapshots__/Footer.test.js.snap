// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component SNAPSHOT 1`] = `
<footer
  class="sc-bdfBwQ jygzMn"
>
  <small>
    Â© 2018-2019
  </small>
</footer>
`;
